pe-parser-library/src/unicode_winapi.cpp

```diff
@@ -1,18 +1,14 @@
 /*
 The MIT License (MIT)
-
```

Comment:
Did you mean to remove these lines?

Reply:
Bleh, that's from a bad revert. Will fix.